src/app/views/schedule/Schedule.scss

```diff
@@ -45,9 +45,10 @@
     color: white;
 
     .schedule-countdown {
-      width:55vw;
+      // width:55vw;
```

Comment:
Could we remove this unused style rather than leaving it commented to reduce clutter?